cmd/cluster/destroy.go

```diff
@@ -42,6 +60,7 @@ func NewDestroyCommand() *cobra.Command {
 		Name:               "",
 		AWSCredentialsFile: "",
 		PreserveIAM:        false,
+		Region:             "us-east-1",
```

Comment:
shouldn't we leave this empty?

Reply:
It's actually required input for this command and I'm not sure everyone realizes it